test/human_readable_gtest.cc

```diff
@@ -0,0 +1,70 @@
+//===---------------------------------------------------------------------===//
+// human_readable_test - Unit tests for human readable converters
+//===---------------------------------------------------------------------===//
+
+#include "../src/string_util.h"
+#include "gtest/gtest.h"
+
+namespace {
+
+TEST(HumanReadableTest, base2) {
+  for (int i = 2; i < 10; ++i) {
+    const auto res = benchmark::Base2HumanReadableFormat(1 << i);
+
+    const std::string expected = "2^" + std::to_string(i);
+    EXPECT_STREQ(res.c_str(), expected.c_str());
+  }
+}
+
+TEST(HumanReadableTest, base10) {
```

Comment:
i'd prefer if this one test was split into a test per case.  having multiple test cases in a single test makes it harder to debug should one of the test cases fail.